res/controllers/American Audio VMS4.midi.xml

```diff
@@ -922,6 +934,18 @@
             <status>0x80</status>
             <midino>0x2E</midino>
          </control>
+         <control><!-- on -->
+            <group>[Channel2]</group>
+            <key>cue_play</key>
```

Comment:
Personally I think it's most useful to map shift + cue to `start_stop`, but I guess this works :man_shrugging: